qdrant-landing/content/documentation/frameworks/openllmetry.md

```diff
@@ -0,0 +1,50 @@
+---
+title: OpenLLMetry
+weight: 2300
+---
+
+# OpenLLMetry
+
+OpenLLMetry from [Traceloop](https://www.traceloop.com/) is a set of extensions built on top of [OpenTelemetry](https://opentelemetry.io/) that gives you complete observability over your LLM application.
+
+OpenLLMetry supports instrumenting the `qdrant_client` Python library and exporting the traces to various observability platforms, as listed [here](https://www.traceloop.com/docs/openllmetry/integrations/introduction).
+
+## Usage
+
+Install the SDK
+
+```console
+pip install traceloop-sdk
+```
+
+Instantiate the SDK
+
+```python
+from traceloop.sdk import Traceloop
+
+Traceloop.init()
```

Comment:
I tried this, and got the following error message:

```
>>> Traceloop.init()
Error: Missing Traceloop API key, go to https://app.traceloop.com/settings/api-keys to create one
Set the TRACELOOP_API_KEY environment variable to the key
```

I tried registering, got an API key. https://app.traceloop.com/trace suggested that I replace this line with:

``` 
Traceloop.init(disable_batch=True, api_key="<my_api_key>")
```

I got about 10 lines of traceback, followed by 

`AttributeError: type object 'QdrantClient' has no attribute 'upload_points'`

Reply:
Resolving. With qdrant-client v1.7.3, this works.